raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
```

Comment:
I like a lot of what has been suggested here. I do wonder if we should rename the doc `Raising-Tickets.md` and replace all mention of `Jira` with `ticket. That way these guidelines are universal regardless of which ticketing system we use currently or in future. Also it helps with jargon, jira is not a think, a ticket is a think jira just happens to be the system we use today. Also it would be easier for others outside our organisation to use these guidelines or folk them (remember this is a public open source repo).